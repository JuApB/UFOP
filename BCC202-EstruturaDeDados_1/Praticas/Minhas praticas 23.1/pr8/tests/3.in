3
34
Bianca 91 3
AnaBeatriz 87 1
Felipe 93 2
MariaFlor 13 2
Theo 77 2
Amanda 83 1
Leticia 11 1
MariaVitoria 83 1
Breno 50 1
Guilherme 64 1
Alice 82 3
MariaFernanda 29 3
Emanuelly 59 2
Lucca 19 3
Allana 51 2
Fernanda 100 1
Mirella 68 1
Clara 62 1
Isabelly 60 2
MariaHelena 40 1
Giovanna 27 1
Gabriela 42 2
Sophie 91 1
Anthony 64 3
Esther 59 1
Joana 90 3
Vicente 48 2
Kaique 66 2
Caio 56 2
Ruan 42 1
Marina 73 3
Luana 60 3
Isabella 34 3
Yuri 42 1
18
Caio 29 3
Victor 17 1
ArthurGabriel 37 3
Lorena 90 3
AnaLivia 84 1
Leonardo 17 2
Luna 90 3
Antonia 83 2
Malu 34 3
Isadora 50 1
Gustavo 92 1
Enrico 53 1
JoaoLucas 48 1
PedroHenrique 40 1
Eduarda 40 2
LuizOtavio 46 1
AnaBeatriz 91 1
Daniel 49 1
45
Bruno 15 1
Allana 53 1
JoaoPedro 66 3
Larissa 36 3
Gabrielly 78 3
LuizOtavio 21 1
Caleb 77 2
Theo 34 2
Ayla 54 2
Nathan 90 2
Sophia 19 3
MariaHelena 49 2
Bento 97 1
MariaClara 27 1
Lucas 66 2
Leonardo 60 2
MariaHeloisa 83 3
Vinicius 64 1
MariaCecilia 30 3
PedroMiguel 73 3
Pietra 92 3
Joana 88 1
EnzoGabriel 94 2
Luiza 19 1
Nicole 10 1
Thiago 96 3
Giovanna 81 3
Davi 70 3
MariaAlice 22 2
Gabriela 67 2
MariaIsis 29 3
Emanuel 34 3
LuizFelipe 78 1
Francisco 89 3
Antonia 59 2
Isadora 87 2
Danilo 79 3
ArthurHenrique 98 1
Benjamin 96 1
Sophie 82 2
Clara 98 1
DaviMiguel 60 1
Catarina 77 2
Laura 86 2
Maria 17 1
