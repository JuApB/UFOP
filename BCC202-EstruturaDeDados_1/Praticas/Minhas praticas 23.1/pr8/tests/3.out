Thiago 96 3
Lorena 90 3
Pietra 92 3
Luna 90 3
ArthurGabriel 37 3
Bianca 91 3
Francisco 89 3
Luana 60 3
Alice 82 3
Antonia 83 2
Catarina 77 2
MariaFernanda 29 3
Joana 90 3
Gabrielly 78 3
MariaHeloisa 83 3
MariaIsis 29 3
Larissa 36 3
Malu 34 3
Caio 29 3
Gabriela 67 2
Theo 77 2
Gabriela 42 2
Ayla 54 2
Felipe 93 2
AnaBeatriz 91 1
Emanuelly 59 2
Giovanna 81 3
Marina 73 3
Davi 70 3
Danilo 79 3
PedroMiguel 73 3
MariaFlor 13 2
Caleb 77 2
Emanuel 34 3
MariaCecilia 30 3
Nathan 90 2
EnzoGabriel 94 2
Breno 50 1
Gustavo 92 1
Leonardo 17 2
Lucas 66 2
Leonardo 60 2
Antonia 59 2
AnaBeatriz 87 1
ArthurHenrique 98 1
Sophie 91 1
Clara 98 1
Joana 88 1
Eduarda 40 2
LuizOtavio 46 1
Esther 59 1
Victor 17 1
Amanda 83 1
Vicente 48 2
Anthony 64 3
Kaique 66 2
Lucca 19 3
Caio 56 2
JoaoPedro 66 3
Ruan 42 1
Allana 51 2
Leticia 11 1
Sophia 19 3
Fernanda 100 1
Bento 97 1
MariaVitoria 83 1
MariaHelena 49 2
Yuri 42 1
Isabella 34 3
Mirella 68 1
Isadora 87 2
Clara 62 1
Sophie 82 2
Isabelly 60 2
Laura 86 2
Luiza 19 1
Nicole 10 1
Bruno 15 1
LuizOtavio 21 1
MariaClara 27 1
MariaHelena 40 1
Guilherme 64 1
MariaAlice 22 2
Giovanna 27 1
LuizFelipe 78 1
Daniel 49 1
Theo 34 2
Isadora 50 1
Vinicius 64 1
AnaLivia 84 1
Benjamin 96 1
Enrico 53 1
Allana 53 1
JoaoLucas 48 1
DaviMiguel 60 1
PedroHenrique 40 1
Maria 17 1
